.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,54 @@
+<!--
+Thank you for your interest in ESLint! Please fill out the appropriate template for your issue below, as this will help us greatly in addressing your issue:
+
+1. Bug Reports
+2. New Rules
+3. Change Requests
+4. For any other questions, please stop by our chatroom: https://gitter.im/eslint/eslint
+
+Note that issues submitted with incomplete information may be closed without comment.
```

Comment:
Simple option: Remove "without comment"

Less simple option: Something like "Note that issues submitted with incomplete information may be closed if requests for more information are not fulfilled."